src/sentry/web/frontend/accounts.py

```diff
@@ -33,6 +33,11 @@
 from sentry.utils.safe import safe_execute
 
 
+def send_confirm_emails(user):
+    for email in user.emails.filter(is_verified=False):
+        email.send_confirm_email()
```

Comment:
Regarding my comment about the `select_related()`, this becomes an `O(n)` with number of unverified emails.